ecs.php

```diff
@@ -132,6 +133,7 @@
             ForbiddenDumpSniff::class => [
                 __DIR__ . '/packages/framework/src/Component/DateTimeHelper/Exception/CannotParseDateTimeException.php',
                 __DIR__ . '/packages/framework/src/Twig/VarDumperExtension.php',
+                __DIR__ . '/packages/framework/src/Model/Security/AccessControl/RouteAccessControlDataProvider.php',
```

Comment:
Is it necessary? I don't see any forbidden functions being used.

Reply:
var_export probably